charts/appdynamics-cluster-agent/20.7.0/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+appVersion: "4.5.16"
+description: Helm chart to deploy AppDynamics ClusterAgent for Kubernetes
+name: appdynamics-cluster-agent
+version: 20.7.0
+home: https://www.appdynamics.com/
+icon: https://raw.githubusercontent.com/Appdynamics/appdynamics-charts/master/logo.png
```

Comment:
Can you make a local copy of the logo, place it one level up in `appdynamics-cluster-agent/`, and reference it here as `file://../logo.png`? This is so that it can be used for airgap environments.